scripts/file_uploader.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+"""
+Commandline script to upload any file to KE-chain
+"""
+from __future__ import print_function
+
+import logging
+import os
+import sys
+
+from envparse import Env
+
+try:
+    from pykechain import Client
+except ImportError:
+    sys.path.append('..')
+    from pykechain import Client
+
+from pykechain.models import AttachmentProperty
+from pykechain.exceptions import NotFoundError, MultipleFoundError, LoginRequiredError
+
+env = Env()
+env.read_envfile('../.env')
```

Comment:
i agree, however this is a standalone script with its own interaction and just uses the pykechain lib as if it was installed as its own lib. let us discuss in person.